web-app/js/portal/details/NCWMSColourScalePanel.js

```diff
@@ -39,6 +39,9 @@ Portal.details.NCWMSColourScalePanel = Ext.extend(Ext.Panel, {
         });
 
         this.items = [
+            new Ext.Spacer({
+                height: 10
+            }),
```

Comment:
I don't think it makes sense to add a spacer to this component. If we want there to be more space around it then that space should really be in the parent's list of items (ie. https://github.com/aodn/aodn-portal/pull/1807/files#diff-8da76661a875615270e9fb526f5efac0R86).